src/main/resources/pdiresources/jobs/migration-job.kjb

```diff
@@ -0,0 +1,446 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
since this is the control center, can we rename it to be something more generic such as: "control-center.kjb"